packages/checkbox-group/test/checkbox-group.test.js

```diff
@@ -564,70 +564,6 @@ describe('vaadin-checkbox-group', () => {
     });
   });
 
-  describe('array mutation methods', () => {
-    let bind;
-
-    beforeEach(async () => {
-      group = fixtureSync(`
-        <dom-bind>
```

Comment:
Please also remove the import for this element.

Reply:
Done.